pkg/kubelet/rkt/rkt.go

```diff
@@ -181,6 +181,7 @@ type Runtime struct {
 }
 
 var _ kubecontainer.Runtime = &Runtime{}
+var _ kubecontainer.DirectStreamingRuntime = &Runtime{}
```

Comment:
This is just asserting that `&Runtime{}` implements the `kubecontainer.DirectStreamingRuntime` interface. Nothing is being assigned.